Automattic/jetpack
# Search: remove masterbar search menu to Calypso search admin page

### The PR could be merged beforehand, however make sure NOT to merge `D84599-code` before #25094, `D84235-code` and new jetpack packages deployed to WPCOM.

#### Changes proposed in this Pull Request:
Remove search submenu from Calypso masterbar to allow (WP-Admin ) Search Dashbaord submenu to be registered. The PR needs to be tested on WPCOM.

We'll (1) merge the PR first and then (2) tweak `D84599-code` to allow Search Dashboard submenu to be added.

#### Other information:


#### Jetpack product discussion
pcNPJE-UW-p2

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* Ensure all GitHub checks pass
* Apply `D84599-code` to your WPCOM sandbox
* Sandbox `public-api.wordpress.com` and a simple site of your choice
* Ensure Search menu is removed in sidebar

|Before      | After    |
|----------|--------|
| ![image](https://user-images.githubusercontent.com/1425433/180130179-aff8b27c-00d3-4233-a824-1358c7f56f29.png) | ![image](https://user-images.githubusercontent.com/1425433/180130378-623ac2ec-39bc-4ebc-945e-5071e23a0aa8.png) |
